### DISCUSSION TITLE
页面隐藏功能

### DISCUSSION THREAD

**Discussion by Yellowwwwwww121** [note286/xduts#59] [Q&A] [answered]
你好楼主，我想要隐藏掉致谢，代码如下：
style/remove-page = {acknowledgements},
但不知为何运行以后隐藏失败，正文后面和目录里面还有致谢。
请问是我代码写错了吗?如果是，请教一下楼主正确写法是什么，非常感谢！

**Comment by note286** [comment]
请认真阅读文档。

![image](https://user-images.githubusercontent.com/16207292/222999850-d292f4ae-0eb1-4bf6-a358-376dd91cda07.png)

**Comment by Yellowwwwwww121** [comment]
![image](https://user-images.githubusercontent.com/126239297/222999877-9ca18604-a15b-42e0-8bda-10690bf4e808.png)
不是很明白括号里填写的内容，还请明示

**Reply by note286** [reply]
还不够清楚吗？？？？

```latex
\xdusetup{style/remove-page={致谢}}
```

**Reply by note286** [reply]
而且，就算有点不确定，在文档最后一页，也有如下内容：

![image](https://user-images.githubusercontent.com/16207292/223001055-90b858e3-5294-4af5-b926-01a40bd6d35a.png)

思考下就明白怎么回事了。

**Comment by note286** [comment, marked as answer]
如果你要进行相似性检测即查重，或许应该移除所有的，即：

```latex
\documentclass{xdupgthesis}
\xdusetup{
style/remove-page={封面,题名页,声明页,摘要,索引,对照表,目录,附录,参考文献,致谢,作者简介},
style/remove-header=true,
style/remove-footer=true
}
\begin{document}
\end{document}
```

**Reply by note286** [reply]
文档中也增加了示例：

![image](https://user-images.githubusercontent.com/16207292/223040428-ec4ced9e-3575-46a2-82a2-ac7f9e02a707.png)
